src/js/components/Form/stories/SolicitedFeedback.js

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  TextArea,
+  Header,
+  Heading,
+  StarRating,
+  RadioButtonGroup,
+  ResponsiveContext,
+} from 'grommet';
+
+// This example shows a way to perform validation across multiple fields.
+export const SolicitedFeedback = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box pad="large" gap="medium" width="medium">
+      <Header
+        direction="column"
+        align="start"
+        gap="xxsmall"
+        pad={{ horizontal: 'xxsmall' }}
+      >
+        <Heading level={2} size="small" margin="none">
+          Let us know how your expirence was!
+        </Heading>
+      </Header>
+      <Box
+        // Padding used to prevent focus from being cutoff
```

Comment:
This seems a bit odd to me. What is cutting the focus off?

Reply:
taken out